src/components/structures/MatrixChat.tsx

```diff
@@ -1007,6 +1020,11 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
         this.themeWatcher.recheck();
     }
 
+    private viewFavouriteMessages() {
+        this.setPage(PageType.FavouriteMessagesView);
```

Comment:
I wonder what method calls from `viewHome()` should also be here - I think we need the logged in view? What about the theme thing - what does that do?

Reply:
As for the logged in view, yea i guess it's needed but regarding the theme thing, i am not too sure what it does. But so far in my demo when i switch themes it works correctly, so i don't think it's necessary in this case.